src/control/xojfile/LoadHandler.cpp

```diff
@@ -378,7 +380,13 @@ void LoadHandler::parseBgPixmap() {
 
         this->page->setBackgroundImage(img);
     } else if (!strcmp(domain, "clone")) {
-        PageRef p = doc.getPage(stoull(filepath.string()));
+        gchar* endptr = nullptr;
+        const gchar* filename = filepath.string().c_str();
+        size_t nr = static_cast<size_t>(g_ascii_strtoull(filename, &endptr, 10));
+        if (endptr == filename) {
```

Comment:
This is dangling, filename doesn't exit anymore, after thar line. Use: 
```suggestion
```suggestion
        auto const& filename = filepath.string();
        size_t nr = static_cast<size_t>(g_ascii_strtoull(filename.c_str(), &endptr, 10));
        if (endptr == filename.c_str()) {
```
        const auto filename = filepath.string();
```

Reply:
Thanks!